src/20230907_AaveV3_Multi_FreezeStewards/FreezeStewards.md

```diff
@@ -0,0 +1,33 @@
+---
+title: "Freeze Stewards"
+author: "BGD Labs"
+discussions: ""
```

Comment:
The post where we comment about it on GHO pausing thread

Reply:
Done!